tests/Functional/BackedEnumPlainResourceTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Tests\Functional;
+
+use ApiPlatform\Metadata\Get;
+use ApiPlatform\Metadata\GetCollection;
+use ApiPlatform\Metadata\Link;
+use ApiPlatform\Symfony\Bundle\Test\ApiTestCase;
+use ApiPlatform\Tests\Fixtures\TestBundle\ApiResource\PlainEnumResource;
+
+final class BackedEnumPlainResourceTest extends ApiTestCase
+{
+    public function testOnlyGetOperationsAddedWhenNonSpecified(): void
+    {
+        $factory = self::getContainer()->get('api_platform.metadata.resource.metadata_collection_factory');
+        $resourceMetadata = $factory->create(PlainEnumResource::class);
+
+        $this->assertCount(1, $resourceMetadata);
+        $resource = $resourceMetadata[0];
+        $operations = iterator_to_array($resource->getOperations());
+        $this->assertCount(2, $operations);
+
+        $this->assertInstanceOf(GetCollection::class, $operations['_api_/plain_enum_resources{._format}_get_collection']);
+        $this->assertInstanceOf(Get::class, $operations['_api_/plain_enum_resources/{value}{._format}_get']);
+    }
+
+    public function testEnumsAreAssignedValuePropertyAsIdentifierByDefault(): void
+    {
+        $linkFactory = self::getContainer()->get('api_platform.metadata.resource.link_factory');
+        $result = $linkFactory->completeLink(new Link(fromClass: PlainEnumResource::class));
+        $identifiers = $result->getIdentifiers();
+
+        $this->assertCount(1, $identifiers);
+        $this->assertNotContains('id', $identifiers);
+        $this->assertContains('value', $identifiers);
+    }
+
+    public function testCollection(): void
+    {
+        self::createClient()->request('GET', '/plain_enum_resources', ['headers' => ['Accept' => 'application/json']]);
```

Comment:
we should do Json-LD tests instead also?

Reply:
Totally! As luck has it they are in #6288, hence this PR depending on that one.